misaligned.c

```diff
@@ -16,6 +33,14 @@ int printColorMap() {
 int main() {
     int result = printColorMap();
     assert(result == 25);
+    // Similary all pairs shall be evaluated using the testColorPair variable
+    assert(testColorPair[0].pairNum == 1);
+    assert(strcmp(testColorPair[0].majorColor, "White") == 0);
+    assert(strcmp(testColorPair[0].minorColor, "Blue") == 0);
+    assert(testColorPair[1].pairNum == 2);
+    assert(strcmp(testColorPair[1].majorColor, "White") == 0);
+    assert(strcmp(testColorPair[1].minorColor, "Orange") == 0);

```

Comment:
The tests should include how color/pair number are printed and their alignment also. Please consider the pair with single and double digits and its effect on alignment.